BigBaseV2/src/views/vehicle/view_lsc.cpp

```diff
@@ -708,9 +718,7 @@ namespace big
 							selected_color = color;
 							interior_color = color;
 
-							g_fiber_pool->queue_job([] {
-								VEHICLE::SET_VEHICLE_INTERIOR_COLOR_(player_vehicle, interior_color);
-							});
+							VEHICLE::SET_VEHICLE_INTERIOR_COLOR_(player_vehicle, interior_color);
```

Comment:
Calling natives outside of script threads is detected.

Use the fiber pool please.